src/core/transform.js

```diff
@@ -25,14 +25,45 @@ import p5 from './main';
  * alt="The transformation matrix used when applyMatrix is called"/>
  *
  * @method applyMatrix
- * @param  {Number|Array} a numbers which define the 2×3 matrix to be multiplied, or an array of numbers
- * @param  {Number} b numbers which define the 2×3 matrix to be multiplied
- * @param  {Number} c numbers which define the 2×3 matrix to be multiplied
- * @param  {Number} d numbers which define the 2×3 matrix to be multiplied
- * @param  {Number} e numbers which define the 2×3 matrix to be multiplied
- * @param  {Number} f numbers which define the 2×3 matrix to be multiplied
+ * @param  {Array} arr an array of numbers
  * @chainable
- * @example
+ * 
+ */
+  
+/**
+ * @method applyMatrix
+ * @param  {Number} a_2x3 numbers which define the 2×3 matrix to be multiplied
+ * @param  {Number} b_2x3 numbers which define the 2×3 matrix to be multiplied
+ * @param  {Number} c_2x3 numbers which define the 2×3 matrix to be multiplied
+ * @param  {Number} d_2x3 numbers which define the 2×3 matrix to be multiplied
+ * @param  {Number} e_2x3 numbers which define the 2×3 matrix to be multiplied
+ * @param  {Number} f_2x3 numbers which define the 2×3 matrix to be multiplied
+ * @chainable
+ * 
+ */
+  
+/**
+* @method applyMatrix
+* @param  {Number} a_4x4 numbers which define the 4x4 matrix to be multiplied
```

Comment:
Rationale:

`p5.RendererGL.applyMAtrix(arguments)` is called from this method which then expects either 6 or 16 values

```js
p5.RendererGL.prototype.applyMatrix = function(a, b, c, d, e, f) {
  if (arguments.length === 16) { // 4x4 matrix
    p5.Matrix.prototype.apply.apply(this.uMVMatrix, arguments);
  } 
};
```